parambokeh/__init__.py

```diff
@@ -29,6 +29,23 @@
     __version__ = '0.2.1-unknown'
```

Comment:
Not sure why we need this conditional anymore. @ceball ?

Reply:
Not quite sure how this is in a pr about label format (I'm on my phone so can't easily see whole context), but isn't this just a case of parambokeh not having been updated to use autover yet?